README.rst

```diff
@@ -72,7 +83,6 @@ Python Version Support
 Rig is tested against the following versions of Python:
 
 * 2.7
-* 3.3
```

Comment:
This should really be done in another commit along with tox/travis changes -- will make a PR with this change shortly.